collectors/ciscoamp/collector.js

```diff
@@ -96,8 +96,8 @@ class CiscoampCollector extends PawsCollector {
         utils.getAPILogs(baseUrl, base64EncodedString, apiUrl, state, [], process.env.paws_max_pages_per_invocation)
             .then(({ accumulator, nextPage, newSince }) => {
                 state.apiQuotaResetDate = null;
-                if (!nextPage && state.stream == Events && newSince) {
-                    state.until = moment(newSince).toISOString();
```

Comment:
Do we do `.toISOString()` conversion somewhere else before it gets serialised into SQS message?

Reply:
Yes i am doing here- https://github.com/alertlogic/paws-collector/pull/304/files#diff-6afff9141604ac4282cb8f121376b92e9886b7225dd70f56ba3b316dec3f22dbR34